api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxNewDataset.java

```diff
@@ -0,0 +1,206 @@
+		    "1<1",
+		    "TX_NEW: New on ART: Patients below 1 year",
+		    new Mapped(patientBelow1YearEnrolledInHIVStartedARTIndicator, ParameterizableUtil
+		            .createParameterMappings("startDate=${startDate},endDate=${endDate}")), "");
+		
+		// Male and Female between 1 and 9 years		
+		CompositionCohortDefinition patientBetween1And9YearsEnrolledInHIVStartedART = compositionCohortQueries
+		        .getPatientBelow1YearEnrolledInHIVStartedART(inARTProgramDuringTimePeriod, patientWithSTARTDRUGSObs,
+		            patientWithHistoricalDrugStartDateObs, transferredFromOtherHealthFacility, PatientBetween1And9Years);
+		CohortIndicator patientBetween1And9YearsEnrolledInHIVStartedARTIndicator = GenericIndicators.newCohortIndicator(
+		    "patientBelow1YearEnrolledInHIVStartedARTIndicator", patientBetween1And9YearsEnrolledInHIVStartedART,
+		    ParameterizableUtil
+		            .createParameterMappings("onOrAfter=${startDate},onOrBefore=${endDate},effectiveDate=${endDate}"));
+		dataSetDefinition.addColumn(
+		    "119",
+		    "TX_NEW: New on ART: Patients Between 1 and 9 years",
+		    new Mapped(patientBetween1And9YearsEnrolledInHIVStartedARTIndicator, ParameterizableUtil
+		            .createParameterMappings("startDate=${startDate},endDate=${endDate}")), "");
+		
+		// Male		
+		int i = 2;
+		for (AgeCohortDefinition ageCohort : agesRange) {
+			CompositionCohortDefinition patientInYearRangeEnrolledInARTStarted = compositionCohortQueries
+			        .getPatientInYearRangeEnrolledInARTStarted(inARTProgramDuringTimePeriod, patientWithSTARTDRUGSObs,
+			            patientWithHistoricalDrugStartDateObs, transferredFromOtherHealthFacility, PatientBetween1And9Years,
+			            ageCohort, males);
+			CohortIndicator patientInYearRangeEnrolledInHIVStartedARTIndicator = GenericIndicators.newCohortIndicator(
+			    "patientInYearRangeEnrolledInHIVStartedARTIndicator", patientInYearRangeEnrolledInARTStarted,
+			    ParameterizableUtil
+			            .createParameterMappings("onOrAfter=${startDate},onOrBefore=${endDate},effectiveDate=${endDate}"));
+			dataSetDefinition.addColumn(
+			    "1M" + i,
+			    "Males:TX_NEW: New on ART by age and sex: " + ageCohort.getName(),
+			    new Mapped(patientInYearRangeEnrolledInHIVStartedARTIndicator, ParameterizableUtil
+			            .createParameterMappings("startDate=${startDate},endDate=${endDate}")), "");
+			
+			i++;
+		}
+		
+		// Female		
+		int j = 2;
+		for (AgeCohortDefinition ageCohort : agesRange) {
+			CompositionCohortDefinition patientInYearRangeEnrolledInARTStarted = compositionCohortQueries
+			        .getPatientInYearRangeEnrolledInARTStarted(inARTProgramDuringTimePeriod, patientWithSTARTDRUGSObs,
+			            patientWithHistoricalDrugStartDateObs, transferredFromOtherHealthFacility, PatientBetween1And9Years,
+			            ageCohort, females);
+			CohortIndicator patientInYearRangeEnrolledInHIVStartedARTIndicator = GenericIndicators.newCohortIndicator(
```

Comment:
@rubailly please extract this out to the `library/indicators` folder same as we did for the DataSets. We can maybe group these indicators by a program, such as `HIVIndicators.java`? They are generic enough that they could be reused in other datasets

Reply:
[Done](https://github.com/esaude/openmrs-module-eptsreports/commit/93a2470685736b6bcde1081b26d22fef4d0a0c1d). I added two libraries: HivIndicators and TbIndicators